localize.js.org

derek-b opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```localize.js.org``` to make it work.
I'll merge after we have "contact". Please keep this PR open.

derek-b commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `localize.js.org` to make it work. I'll merge after we have "contact". Please keep this PR open.

Thank you.  I have added the CNAME to the correct location in our repository now.

indus commented:
Welcome https://localize.js.org
